Rimokon/import_config.py

```diff
@@ -0,0 +1,145 @@
+"""
+This file imports the user-defined config.py file and processes all the entries to
+bring them to the common form and make it easy to use them.
```

Comment:
```suggestion
bring them to the common form and make it easy to use them further in the rest of the code.
```
, or something like that